ember-cli-build.js

```diff
@@ -1,13 +1,19 @@
 'use strict';
-
+const path = require('path');
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
+const MATERIALIZE_SASS_FOLDER = path.join(
+  require.resolve('materialize-css'),
+  '..',
+  '..',
+  '..',
+  'sass'
+);
+
```

Comment:
This is ugly. Maybe this library should provide a function for calculating the Sass path?

Reply:
I believe the `sass` path is specified in their `package.json`. Wouldn't it make the most sense to get it from there so they can move it wherever they like without breaking anything?
https://github.com/Dogfalo/materialize/blob/v1-dev/package.json#L9

Reply:
What about something like this?
```
'use strict';
const { dirname, resolve } = require('path');
const basePath = dirname(require.resolve('materialize-css/package.json'));
const relativeSassPath = dirname(require('materialize-css/package.json').sass);
const fullSassPath = resolve(basePath, relativeSassPath);
console.log(`basePath = ${basePath}\r\nrelativeSassPath = ${relativeSassPath}\r\nfullSassPath = ${fullSassPath}`);
/* Output:
basePath = /project-path/node_modules/materialize-css
relativeSassPath = sass
fullSassPath = /project-path/node_modules/materialize-css/sass
*/
```